org-gtd-review.el

```diff
@@ -142,6 +142,23 @@ mostly of value for testing purposes."
                         (org-agenda-skip-additional-timestamps-same-entry t))))))))
         (org-agenda nil "g"))))
 
+(defun org-gtd-review-missed-items ()
+  "Agenda view with all incubated or delegated actions whose date is in the past."
+  (interactive)
+  (with-org-gtd-context
+      (let ((org-agenda-custom-commands
+             '(("g" "foobar"
+                ((tags "ORG_GTD=\"Incubated\"+ITEM<>\"Incubate\"|"
```

Comment:
This is kind of ugly, but I need to filter out the ``"Incubate"`` heading, otherwise I skip this heading completely